themes/default/content/blog/deprecating-non-project-mode/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Deprecating non-project mode from DIY backends"
+date: 2024-02-19
+meta_desc: Read about the plans for the removal of non-project mode.
+authors:
+    - fraser-waters
+tags:
+    - features
+    - pulumi-releases
+---
+
+Last year we released [project support for self-managed backends](/blog/project-scoped-stacks-in-self-managed-backend/), we now plan to remove support for non-project mode.
+
+<!--more-->
+
+Users using a DIY backend (previously referred to as self-managed) may have seen a warning in recent releases of Pulumi:
```

Comment:
Just wondering -- have we renamed these from "self-managed backends" to "DIY backends"?

Reply:
Yes